### PR TITLE
Init Nodejs sample for Device endpoint

### PR DESCRIPTION
**Pull request by hminh21**
@NguyenHoanTien cc. @khanhdodang, please help me review this PR. Thank you!

### CODE REVIEW

File: kobiton-rest-api/Nodejs/Device/fetch-available-auto-devices.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    json: true,
+    method: 'GET',
+    auth: {
+        user: username,
+        pass: apiKey
+    },
+    headers,
+    qs: {
+        groupId,
+        isOnline: 'true',
```
Comment (khanhdodang) [inline kobiton-rest-api/Nodejs/Device/fetch-available-auto-devices.js]: this is boolean value

File: kobiton-rest-api/Nodejs/Device/get-all-devices.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import request from 'request'
+import {headers, username, apiKey, groupId} from '../config'
+
+if(!username || !apiKey)
+{
+    console.log("KOBITON_USERNAME and KOBITON_APIKEY variables are need to execute the script")
```
Comment (khanhdodang) [inline kobiton-rest-api/Nodejs/Device/get-all-devices.js]: Use single quote

File: kobiton-rest-api/Nodejs/Device/fetch-android-7.0-devices.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import request from 'request'
+import {headers, username, apiKey, groupId} from '../config'
+
+if(!username || !apiKey)
+{
+    console.log("KOBITON_USERNAME and KOBITON_APIKEY variables are need to execute the script")
+    process.exit(1)
+}
+
+request({
```
Comment (khanhdodang) [inline kobiton-rest-api/Nodejs/Device/fetch-android-7.0-devices.js]: Could we reuse these methods?

Comment (hminh21) [reply kobiton-rest-api/Nodejs/Device/fetch-android-7.0-devices.js]: I think we couldn't

Comment (hminh21) [reply kobiton-rest-api/Nodejs/Device/fetch-android-7.0-devices.js]: I think we couldn't

**Pull request closed without merging**